style.css

```diff
@@ -0,0 +1,94 @@
+* {
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #eee;
+}
+
+#header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    position: fixed;
+    width: auto;
+    background-color: #eee;
+}
+
+#header>img {
+    width: 30%;
+}
+
+#features >div {
+    margin: 2rem;
+}
+
+#nav-bar>ul {
+    list-style: none;
+    display: flex;
+    gap: 2rem;
+}
+
+.nav-link {
+    text-decoration: none;
+}
+
+footer {
+    background-color: #ddd;
+}
+
+#how-it-works {
+    width: 300px;
+    margin: auto;
+}
+
+#pricing {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+#pricing>div>.head {
```

Comment:
This style and the one on line 39 share the same attribute and value. While this works, it is recommended that you group all targeted elements with similar styling to avoid repetitions. Please try to group them together.

Reply:
> This style and the one on line 39 share the same attribute and value. While this works, it is recommended that you group all targeted elements with similar styling to avoid repetitions. Please try to group them together.

- [ ] I would kindly suggest you to phrased the comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion:kindly consider This style and the one on line 39 share the same attribute and value. While this works, it is recommended that you group all targeted elements with similar styling to avoid repetitions. Please try to group them together.